ambry-tools/src/main/java/com.github.ambry/store/DumpIndexTool.java

```diff
@@ -51,10 +51,15 @@
  * Supported operations are
  * 1. Dump index.
  * 2. Dump an index segment.
- * 3. Verify an index for sanity (no duplicate records).
+ * 3. Verify an index for sanity (no duplicate records and that all index segments are deserializable).
  */
 public class DumpIndexTool {
 
+  /**
+   * Contains information for a single blob regarding
```

Comment:
minor. Can you add info that each entry contains state across all replicas that is fed in

Reply:
Not sure what you mean. There are no replicas here. This is the state of the blob in a single index.

Reply:
Have updated the documentation of `ReplicationStatus` where I think you want this information to be